Tools/autotest/arducopter.py

```diff
@@ -1157,6 +1157,7 @@ def TakeoffCheck(self):
             'SIM_ESC_TELEM': 1,
             'SIM_ESC_ARM_RPM': 500,
             'TKOFF_RPM_MIN': 1000,
+            'TKOFF_RPM_MAX': 2000,
```

Comment:
This now only tests the case when both are set which I think is less likely. Would prefer a new test for the new parameter.

Reply:
This was duplicated below. Does that work, or did you have something else in mind?